tests/unit/ui/components/TestShowMoreCard.js

```diff
@@ -57,4 +58,19 @@ describe('<ShowMoreCard />', () => {
     const rootNode = findDOMNode(root);
     expect(rootNode.textContent).toContain('Hello I am description');
   });
+
+  it('executes truncateToMaxHeight when it recieves props', () => {
+    const node = document.createElement('div');
+    let ref;
+    ReactDOM.render(<ShowMoreCardBase i18n={getFakeI18nInst()} ref={(e) => { ref = e; }} />, node,
+      () => {
+        jest.spyOn(ref, 'truncateToMaxHeight');
+        ReactDOM.render(<ShowMoreCardBase
```

Comment:
The component has been rerendered to call the function again. The reference of first call is bound to spy and then it can actually spy the method on second call.
I looked various ways to achieve this but found nothing better for redux component.